pkg/controller/controller_utils.go

```diff
@@ -703,6 +705,13 @@ func (s ActivePods) Len() int      { return len(s) }
 func (s ActivePods) Swap(i, j int) { s[i], s[j] = s[j], s[i] }
 
 func (s ActivePods) Less(i, j int) bool {
+	// 0. Pod with label 'TO_DELETE=true' is smaller
+	_, exists1 := s[i].Labels[POD_DELETE_LABEL]
```

Comment:
For most cases, we could get those pods in a list, then it is easy to label them by script, e.g. `cat pod_list | xargs -I{} kubectl label pod {} POD_DELETE_LABEL=TRUE`.